qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -26,14 +26,18 @@
             blk_num = 1
             repeat_times = 3
         - multi_pci:
-            blk_num = 2
+            blk_num = 3
             repeat_times = 3
-            images += " stg1"
+            images += " stg1 stg2"
             boot_drive_stg1 = no
+            boot_drive_stg2 = no
             image_name_stg1 = images/storage1
-            image_size_stg1 = 1G
+            image_name_stg2 = images/storage2
+            image_size_equal = 1G
             remove_image_stg1 = yes
+            remove_image_stg2 = yes
             force_create_image_stg1 = yes
+            force_create_image_stg2 = yes
```

Comment:
why not add stg2 in other variants,such as 
 - block_virtio:
- fmt_qcow2:

Reply:
Thanks for question, tested with qcow2 format with this code, the result is match currently requirement, but it seems not dissonance, will  optimization the code, thanks~